docs/Reporting.md

```diff
@@ -1,58 +1,66 @@
 # FCA: Reporting
 
-TODO: add intro
+FCA is pre-configured to generate detailed mochawesome JSON reports following the completion of [sanity tests](./Execution.md). This functionality not only provides users with immediate test results but also enables sending these reports to AWS or any other URL endpoint for further processing and analysis. Reporting needs will vary on a case-by-case basis and users have the flexibility to utilize the generated mochawesome report JSON in various ways, including transforming the JSON data into an HTML report.
 
 ## Table of Contents
 
-- [Supported Ways of Retrieving Reports](#supported-ways-of-retrieving-reports)
-- [Supported Report Parameters](#supported-report-parameters)
+- [Using the Existing Reporting Logic](#using-the-existing-reporting-logic)
+- [Creating Your Own Reporting Logic](#creating-your-own-reporting-logic)
 
-## Supported ways of retrieving reports
-
-EXISTING CONTENTS(needs revision):
-```
-- From UI
-
-  - After executing suite from the UI in FCA, the mocha json will upload to S3.
-  - Download the `json report` from S3 and convert it to `html report`
-    - `arn:aws:s3:::your-s3-Bucket-name`
-  - Steps to convert into `html report`
-    1.  Move the `json report` to root folder
-    2.  Run the command `yarn marge filename.json -f report -o /destination_path`
-    3.  Check the `html report` on the destination path.
-```
+## Using the Existing Reporting Logic
 
 ### Background
 
-TODO: add details
+The default reporting mechanism in FCA is designed to streamline your test analysis process. Post-execution of sanity tests, the app generates detailed mochawesome JSON reports. These reports are crucial for reviewing test outcomes. The `northBoundSchemaValidationAndReportGeneration()` function in [Test_Runner.js](../src/Test_Runner.js) orchestrates this process. Additionally, for users leveraging `IntentReader`, the [runTestHandler](../src/pubsub/handlers/RunTestHandler.js) class can be utilized to execute a sanity test and obtain a reports.
+
+#### Standalone Mode
+
+Furthermore, FCA supports `standalone` report generation. Standalone reporting posts to a different endpoint which allows for more diverse report management and usage.
+
+In order to activate `standalone` mode, include the following paramaters:
```

Comment:
This reads more like setup instructions to me.
Wondering if there is enough standalone specific information to have it's own section instead of including it in `Using the Existing Reporting Logic`.

Reply:
Yes decided to divide into three sections (standard, standalone, & create your own)